deploy/iqe-trigger-integration.yml

```
@@ -117,13 +117,16 @@ parameters:
   from: "[a-z0-9]{6}"
 - name: IQE_IMAGE
   description: "container image path for the iqe plugin"
-  value: quay.io/cloudservices/iqe-tests:hms-integration
+  value: quay.io/cloudservices/iqe-tests
```

```suggestion
  value: quay.io/cloudservices/iqe-tests:insights-experiences
```
Same as in image-builder repo, let's add the tag here.